randovania/gui/data_editor.py

```diff
@@ -355,7 +354,7 @@ def _on_click_link_to_other_node(self, link: str):
             self.focus_on_region_by_name(region_name)
             self.focus_on_area_by_name(area_name)
             if node_name is None:
-                node_name = self.current_area.default_node
+                node_name = self.current_area.nodes[0]
```

Comment:
What is/was the use case of that?
Clicking a link to another node but the node has no name sounds kinda weird to me. I was about to dropping this case completly.

Reply:
It's for having a link to an Area, not a Node.
For sure the teleporter description had these links, dunno if anything else.

Reply:
Found only one `href` using node in this file. The other one was gone with the `TeleporterNode` => removed the check